.drone.jsonnet

```diff
@@ -644,6 +644,13 @@ local integration_cloud_images = Step('cloud-images', depends_on=[integration_im
 
 local integration_reproducibility_test = Step('reproducibility-test', target='reproducibility-test', depends_on=[load_artifacts], environment={ IMAGE_REGISTRY: local_registry });
 
+local integration_siderolink = Step('e2e-siderolink', target='e2e-qemu', privileged=true, depends_on=[load_artifacts], environment={
```

Comment:
it should `depends_on` on the previous set (default-hostname in this case)

Reply:
Done.